### ISSUE TITLE
Create assistance script for assessment

### ISSUE THREAD

**Issue by Ainali** [publiccodenet/standard#996]
While it's not plausible that we in any near-time can create a workflow or a continuous integration template that would automatically create a full assessment, there might be a few things that a script could help us discover.

For example, we could search for commonly used file names like `README`, `CONTRIBUTING`, `GOVERNANCE`, `LICENSE` etc. and link to them from the links and notes field where they might be relevant. If it is a GitHub repository, we could answer some requirements about the version control system and statistics issues and pull requests and possibly even about branch protection. All of these would have to be manually checked, but it would save time just finding suggested files and creating the HTML link.

I suggest we experiment with just a few and see how easy it is and how error-prone it might be, and then iterate on that if succesful.

**Comment by ericherman** [comment]
Yes, and while we'd never make it perfect, this could really save us time at the start of an assessment.
